app/models/user.rb

```diff
@@ -488,6 +488,35 @@ def remove_from_queue
     invite_request.destroy if invite_request
   end
 
+  # Run a set of corrections on a user
+  def check
+
+    # Delete any subscriptions the user has to deleted items because this causes
+    # the user's subscription page to error
+    @subscriptions = self.subscriptions.includes(:subscribable)
+    @subscriptions.to_a.each do |sub|
+      if sub.name.nil?
+        sub.destroy
+      end
+    end
+
+    # AKA "stats page unfucking". It is most often caused by the existence of works with nil revised_at dates.
```

Comment:
I'd suggest something a little more meaningful to future coders than "stats page unfucking", like `Fix user stats page error caused by the existence of works with nil revised_at dates`